### PR TITLE
AutoMapper Implemented

### PR DESCRIPTION
**Pull request by odinaabduganiyeva03**
#28

### CODE REVIEW

File: FleetFlow.Service/Mappers/MapperProfile.cs
Comment (muxammad) [inline FleetFlow.Service/Mappers/MapperProfile.cs:1]: empty classes I think u mde PR without saving

Comment (muxammad) [inline FleetFlow.Service/Mappers/MapperProfile.cs:1]: Empty mapper class u added please re-make PR and check your automapper

Comment (komron2052) [reply FleetFlow.Service/Mappers/MapperProfile.cs:1]: I think that's correct, every person will add your own mappers there

Comment (dotnetgoo) [reply FleetFlow.Service/Mappers/MapperProfile.cs:1]: Conflicts, resolve it

**Pull request closed without merging**